src/modules/shared/services/notifications.service.ts

```diff
@@ -105,6 +105,9 @@ export class NotificationsService extends CoreService {
                 case NotificationContextDetailEnum.COLLABORATOR_INVITE:
                   link = { label: 'Click to go to innovation', url: `/${this.userUrlBasePath()}/innovations/${item.innovation.id}/collaborations/${item.contextId}` };
                   break;
+                case NotificationContextDetailEnum.COLLABORATOR_UPDATE:
```

Comment:
Didn't understand quite well why you have two cases (COLLABORATOR_INVITE and COLLABORATOR_UPDATE) that end up doing the same thing, but I imagine there's some rule on the BE

Reply:
They come from two different handlers on the backend. COLLABORATOR_INVITE is when the invite is created and COLLABORATOR_UPDATE is when the invite is changed (cancelled, accepted, etc.)